API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -101,30 +99,240 @@ However, until the filtering module is written, the geospatial module retains ru
         }
       ]
     }
-  }</pre>
+  }
+  </pre>
   <h4 class="rulelab">How to test</h4>
   <p>Validate that the returned document contains the expected <code>type</code> property for each member.</p>
 </div>
 
 ## Result (response)
 
-In a REST API that uses JSON as the data format, the geometry is returned as a GeoJSON Geometry object.
+In case a REST API shall comply to the OGC API Features specifications, e.g. for usage in GIS applications, the following applies.
+
+<div class="rule" id="api-geo-1">
+  <p class="rulelab"><strong>API-GEO-1</strong>: Support GeoJSON for geospatial APIs</p>
+  <p>For representing 2D geometric information in an API, use the convention for describing geometry as defined in the GeoJSON format [[rfc7946]]. Support GeoJSON as described in OGC API Features <a href="https://docs.ogc.org/is/17-069r3/17-069r3.html#_requirements_class_geojson">Requirements class 8.3</a> [[ogcapi-features-1]]. </p>
+  Example: feature
+  <pre>
+  {
+    "type": "Feature",
+    "id": "0308100000022041",
+    "geometry":  {
+      "type": "Point",
+      "coordinates": [5.2795,52.1933]
+    },
+    "properties": {
+      "naam": "Paleis Soestdijk",
+      ...
+    },
+    "links": [
+    {
+      "self": ""
+    } 
+    ]
+  }
+  </pre>
+
+  Example: feature collection
+  <pre>
+  {
+    "type": "FeatureCollection",
+    "features": [
+    {
+      "type": "Feature",
+      "id": "0308100000022041",
+      "geometry":  {
+        "type": "Point",
+        "coordinates": [5.2795,52.1933]
+      },
+      "properties": {
+        "naam": "Paleis Soestdijk",
+        ...
+      },
+      "links": [
+      {
+        "self": ""
+      } 
+      ]
+    }],
+    "timeStamp" : "2023-02-22T10:32:23Z",
+    "numberMatched" : "0308100000022041",
+    "numberReturned" : "1",
+    "links": [
+    {
+      "self": ""
+    } ,
+    {
+      "next": ""
+    } ,
+    ]
+  }
+  </pre>
+  <p>
+  Note that:
+  
+  - The resources' properties (e.g. <code>naam</code>) are passed in the properties object. Depending on the implemented filter capabilities the properties object may contain all or a selection of the resources' properties.
+  - The OGC API Fearures specifications provides the possibility to add an array of links to a feature and feature collection, which may contain a self link and in case of a feature collection may contain navigation links.
```

Comment:
```suggestion
  - The OGC API Features specification provides the possibility to add an array of links to a feature and feature collection, which may contain a self link and in case of a feature collection may contain navigation links.
```